packages/run-it/src/components/ConfigForm/ConfigForm.tsx

```diff
@@ -139,34 +200,61 @@ export const ConfigForm: FC<ConfigFormProps> = ({
           <Fieldset legend="Server locations">
             <FieldText
               required
-              label="API server url"
+              label="API server URL"
               placeholder="typically https://myserver.looker.com:19999"
               name="baseUrl"
               defaultValue={fields.baseUrl}
               onChange={handleUrlChange}
             />
             <FieldText
-              required
-              label="Auth server url"
-              placeholder="typically https://myserver.looker.com:9999"
-              name="lookerUrl"
-              defaultValue={fields.lookerUrl}
-              onChange={handleUrlChange}
+              label="Auth server URL"
+              placeholder="Click 'Verify' to retrieve"
+              name="webUrl"
+              defaultValue={fields.webUrl}
+              disabled={true}
             />
           </Fieldset>
         </Form>
+        {fields.fetchError && (
+          <>
+            <Text color="danger">{fields.fetchError}</Text>
```

Comment:
I assume this should be critical

Reply:
it is now! And actually changed to a MessageBar